src/setup_payload/Base38.cpp

```diff
@@ -128,11 +127,11 @@ static inline CHIP_ERROR decodeChar(char c, uint8_t & value)
         kBogus, // '(', =40
         kBogus, // ')', =41
         kBogus, // '*', =42
-        37,     // '+', =43
+        kBogus, // '+', =43
         kBogus, // ',', =44
-        38,     // '-', =45
-        39,     // '.', =46
-        40,     // '/', =47
+        36,     // '-', =45
```

Comment:
Could we start this array from `-` and modify conditions and arithmetics below (e.g. subtract 45 rather than 32)? This would avoid allocating many `kBogus` items at the start?

Reply:
Sure, good idea, thanks